anyway/infographics_utils.py

```diff
@@ -914,16 +888,16 @@ def percentage_accidents_by_car_type(involved_by_vehicle_type_data):
         for item in involved_by_vehicle_type_data:
             vehicle_type, count = item["involve_vehicle_type"], int(item["count"])
             total_count += count
-            if vehicle_type in BE_CONST.CAR_VEHICLE_TYPES:
-                driver_types["רכב פרטי"] += count
-            elif vehicle_type in BE_CONST.LARGE_VEHICLE_TYPES:
-                driver_types["מסחרי/משאית"] += count
-            elif vehicle_type in BE_CONST.MOTORCYCLE_VEHICLE_TYPES:
-                driver_types["אופנוע"] += count
-            elif vehicle_type in BE_CONST.BICYCLE_AND_SMALL_MOTOR_VEHICLE_TYPES:
-                driver_types["אופניים/קורקינט"] += count
+            if vehicle_type in VehicleCategory.CAR.get_codes():
+                driver_types[VehicleCategory.CAR.value] += count
+            elif vehicle_type in VehicleCategory.LARGE.get_codes():
+                driver_types[VehicleCategory.LARGE.value] += count
+            elif vehicle_type in VehicleCategory.MOTORCYCLE.get_codes():
+                driver_types[VehicleCategory.MOTORCYCLE.value] += count
+            elif vehicle_type in VehicleCategory.BICYCLE_AND_SMALL_MOTOR.get_codes():
+                driver_types[VehicleCategory.BICYCLE_AND_SMALL_MOTOR.value] += count
             else:
-                driver_types["אחר"] += count
+                driver_types[VehicleCategory.OTHER.value] += count
```

Comment:
I know it's not your code, but what do you think about doing it as follows?
```
for category in [VehicleCategory.CAR, VehicleCategory.CAR, .., other]
    if vehicle_type in category.get_codes():
           driver_types[VehicleCategory.CAR.value] += count
           break
```

where the else category can be a all_possible_values - all_categories

Reply:
but totally optional

Reply:
The `OTHER` category is not the complement of the categories here. (There are additional categories, and other is the rest of the whole group.